src/rtapi/uspace_rtapi_app.cc

```diff
@@ -56,6 +56,10 @@
 #include <sys/shm.h>		/* shmget() */
 #include <string.h>
 
+static int rtapi_clock_nanosleep(clockid_t clock_id, int flags,
```

Comment:
It's needed, because uspace_common.h is included here, but not until the last line of the file(!).  Rather than decypher and correct why that happens to be the case, I put a forward declaration.